authnz/src/main/java/io/stargate/auth/AuthenticationService.java

```diff
@@ -15,11 +15,19 @@
  */
 package io.stargate.auth;
 
+import io.stargate.db.Authenticator.SaslNegotiator;
+
 public interface AuthenticationService {
 
   String createToken(String key, String secret) throws UnauthorizedException;
 
   String createToken(String key) throws UnauthorizedException;
 
   StoredCredentials validateToken(String token) throws UnauthorizedException;
+
+  SaslNegotiator getSaslNegotiator(
+      AuthenticationService authentication,
```

Comment:
Is `authentication` not the same as `this`?

Reply:
True. I'll remove that param.